src/main/java/appeng/client/gui/AEBaseGui.java

```diff
@@ -208,6 +213,22 @@ protected void drawTooltip( int x, int y, List<String> lines )
 		this.drawHoveringText( lines, x, y, this.fontRenderer );
 	}
 
+	@Override
+	protected void renderHoveredToolTip( int mouseX, int mouseY )
+	{
+		Slot slot = this.getSlot( mouseX, mouseY );
+		if( slot != null && slot.isEnabled() && slot instanceof ISlotFluid )
+		{
+			ISlotFluid fluidSlot = (ISlotFluid) slot;
+			if (fluidSlot.getFluidInSlot() != null && fluidSlot.shouldRenderAsFluid()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)